sheetsync/sheetsync/sheets.py

```diff
@@ -273,6 +274,9 @@ def __init__(self, client, sheet_id, worksheets, bustime_start, edit_url, alloca
 		self.bustime_start = bustime_start
 		self.edit_url = edit_url
 
+		# fallback to no shifts if there is a shift parsing error
+		self.latest_shifts = {'repeating':[], 'one_off':[], 'timezone':zoneinfo.ZoneInfo('UTC')}
```

Comment:
I think it would be better to fail entirely (and crash) if the initial parse fails, since this more likely represents a config error. And we don't want a misconfiguration to result in changing ALL videos to remove existing shifts.
```suggestion
		self.latest_shifts = common.shifts.parse_shifts(self.shifts)
```